res/values/strings.xml

```diff
@@ -413,6 +425,9 @@
     <string name="pref_notifications">Notifications</string>
     <string name="pref_notifications_show">Show</string>
     <string name="pref_notifications_priority">Priority</string>
+    <string name="pref_notifications_explain">Enable system notifications on new messages</string>
```

Comment:
"for new messages" sounds nicer to me personally, but would be nice to have a native-speaker review